agences-regionales-sante/saint-martin/2020-03-22.yaml

```
@@ -0,0 +1,11 @@
+date: "2020-03-22"
+source:
+  nom: ARS Guadeloupe
+  url: https://www.guadeloupe.ars.sante.fr/system/files/2020-03/22032020%20st%20martin%20point%20de%20situation%20Coronavirus.pdf
+  archive: https://web.archive.org/web/20200325095130/https://www.guadeloupe.ars.sante.fr/system/files/2020-03/22032020%20st%20martin%20point%20de%20situation%20Coronavirus.pdf
+donneesRegionales:
+  nom: Saint-Martin
+  code: COM-978
+  casConfirmes: 3
```

Attention leur communiqué n'est pas top. Il ne prend pas en compte la totalité des cas confirmés. Ils enlèvent les guéris.
C'est assez prise de tête ( voir les fichiers précédents)

---

Ok donc logiquement les chiffres restent les mêmes alors ?
```suggestion
  casConfirmes: 5
  hospitalises: 1
  reanimation: 0
```